lib/exe/js.php

```diff
@@ -173,7 +173,7 @@ function js_load($file){
         // is it a include_once?
         if($match[1]){
             $base = utf8_basename($ifile);
-            if($loaded[$base]){
+            if(array_key_exists($base, $loaded) && $loaded[$base] === true){
```

Comment:
Empty would do the trick also, I just prefer strict checking, if thats over done, I'll replace it with a !empty... Let me know what you guys prefer :)